front/src/applications/operationalStudies/components/ManageTrainSchedule/PowerRestrictionSelector.tsx

```diff
@@ -38,37 +56,78 @@ export default function PowerRestrictionSelector() {
     }
   };
```

Comment:
This function can be simplified too, we can remove the 2 ifs
```suggestion
    const powerRestrictionsList = Object.entries(rollingStockInfo.power_restrictions).map(
      ([key, _]: [string | undefined, string]) => ({ key, value: key })
    );
    powerRestrictionsList.unshift({ key: undefined, value: t('noPowerRestriction') });
    setPowerRestrictions(powerRestrictionsList);
```

Reply:
Done